frontend/packages/citizen-frontend/src/applications/editor/verification/ContactInfoSection.tsx

```diff
@@ -0,0 +1,146 @@
+            : t.applications.editor.verification.no}
+        </span>
+
+        {formData.childFutureAddressExists && (
+          <>
+            <Label>{tLocal.child.addressChangesAt}</Label>
+            <span>{formData.childMoveDate}</span>
+
+            <Label>{tLocal.child.newAddress}</Label>
+            <span>
+              {formData.childFutureStreet} {formData.childFuturePostalCode}{' '}
+              {formData.childFuturePostOffice}
+            </span>
+          </>
+        )}
+      </ListGrid>
+
+      <Gap size="m" />
+
+      <H3>{tLocal.guardian.title}</H3>
+      <ListGrid
+        labelWidth={ApplicationDataGridLabelWidth}
+        rowGap="s"
+        columnGap="L"
+      >
+        <Label>{tLocal.guardian.name}</Label>
+        <span>
+          {formData.guardianFirstName} {formData.guardianLastName}
+        </span>
+      </ListGrid>
+
+      {type !== 'CLUB' && (
+        <>
+          <Gap size="m" />
+          <H3>{tLocal.secondGuardian.title}</H3>
+          {type === 'DAYCARE' && <ContactInfoSecondGuardianDaycare />}
+          {type === 'PRESCHOOL' && !!formData.otherGuardianAgreementStatus && (
+            <ContactInfoSecondGuardianPreschool formData={formData} />
+          )}
+
+          {showFridgeFamilySection && (
+            <>
+              <Gap size="m" />
+              <H3>{tLocal.fridgePartner.title}</H3>
+              {formData.otherPartnerExists ? (
+                <ListGrid
+                  labelWidth={ApplicationDataGridLabelWidth}
+                  rowGap="s"
+                  columnGap="L"
+                >
+                  <Label>{tLocal.fridgePartner.name}</Label>
+                  <span>
+                    {formData.otherPartnerFirstName}{' '}
+                    {formData.otherPartnerLastName}
+                  </span>
+
+                  <Label>{tLocal.fridgePartner.ssn}</Label>
+                  <span>{formData.otherPartnerSSN}</span>
+                </ListGrid>
+              ) : (
+                <span>{t.applications.editor.verification.no}</span>
+              )}
+
+              <Gap size="m" />
+              <H3>{tLocal.fridgeChildren.title}</H3>
+              {formData.otherChildren.length > 0 ? (
+                formData.otherChildren.map(
+                  ({ firstName, lastName, socialSecurityNumber }) => (
+                    <ListGrid
+                      key={socialSecurityNumber}
+                      labelWidth={ApplicationDataGridLabelWidth}
+                      rowGap="s"
+                      columnGap="L"
+                    >
+                      <Label>{tLocal.fridgeChildren.name}</Label>
+                      <span>
+                        {firstName} {lastName}
+                      </span>
+
+                      <Label>{tLocal.fridgeChildren.ssn}</Label>
+                      <span>{socialSecurityNumber}</span>
+                    </ListGrid>
+                  )
+                )
+              ) : (
```

Comment:
Gaps/grouping between multiple children is a bit off 
![Screenshot 2021-02-03 at 16 31 43](https://user-images.githubusercontent.com/1533854/106763089-284c6480-663f-11eb-803c-9b104eea0362.png)